src/builder-git.c

```diff
@@ -396,10 +396,8 @@ git_mirror_submodules (const char     *repo_location,
 
           lines = g_strsplit (g_strstrip (ls_tree), "\n", 0);
           if (g_strv_length (lines) != 1)
-            {
-              g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "Not a gitlink tree: %s", path);
-              return FALSE;
-            }
+            // There can be path in the gitmodules files that are not submodules.
+            continue;
```

Comment:
Just minor nits: Try to follow the style around this.

```c
/* There can be a path in the gitmodules files thare are not submodules. */
if (g_strv_length (lines) != 1)
  continue;
```

Reply:
done (and fixed grammar).